src/jinja2/runtime.py

```diff
@@ -817,6 +817,10 @@ def __iter__(self):
         if 0:
             yield None
 
+    async def __aiter__(self):
```

Comment:
Would `async def` be a blocker for getting this merged into 2.11.x?And should be merged into master instead?